### DISCUSSION TITLE
Hello from Cadhub 👋

### DISCUSSION THREAD

**Discussion by Irev-Dev** [codesandbox/codesandbox-client#5796] [General] [answered]
Hey,

@franknoirot and myself are working on something completely different to CodeSandbox, but we're drawing a lot of inspiration from your IDE and we just wanted to say hi.

We're building a community hub for a genre of 3d CAD modeling, "Code-CAD" (OpenSCAD is a popular Code-CAD package if you're familiar with it). So completely outside of the javascript eco-system, but we absolutely love how well you've focused on the developer experience. We are still yet to tackle multi-file editing or pulling code from github repos but it's on the todo list.

We did briefly consider forking your client, but it seems pretty well tied into the javascript world (which makes sense) so we're building our own.

Like I said just wanted to say hi, if you're interested here are a couple of links:
- [Github Repo](https://github.com/Irev-Dev/cadhub)
- [OpenSCAD integration WIP](https://cadhub.xyz/dev-ide/openScad)
- [CadQuery integration WIP](https://cadhub.xyz/dev-ide/cadQuery)

Cheers, Kurt.

**Comment by smhutch** [comment, marked as answer]
Really nice project @Irev-Dev! Good luck 🍀 😄
